### PR TITLE
Adding BUFFALO_PLUGIN_TIMEOUT documentation.

### PR DESCRIPTION
**Pull request by marstr**
Fixing up the documentation to account for https://github.com/gobuffalo/buffalo/pull/1104

### CODE REVIEW

**Comment by marstr** [comment]
@lmazuel, would you mind taking a look at the French text I included here? @salameer and I took a stab at your language, but it's probably terrible.

File: templates/docs/config-vars.fr.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ Les variables d'environnement sont une bonne méthode pour séparer les valeurs
...
-| `GO_ENV`              | `development`            | L'« environment » d'exécution de Buffalo                   |
-| `GO_BIN`              | `go`                     | Le compilateur Go à utiliser                               |
-| `BUFFALO_PLUGIN_PATH` | `$PATH`                  | L'endroit où Buffalo va chercher ses plugins               |
-| `ADDR`                | `127.0.0.1` or `0.0.0.0` | L'adresse d'écoute du serveur HTTP                         |
-| `PORT`                | `3000`                   | Le port d'écoute du serveur HTTP                           |
-| `HOST`                | `http://127.0.0.1:$PORT` | L'« URL » de l'application (c-à-d son adresse publique)    |
-| `SESSION_SECRET`      | `""`                     | Un sel utilisé pour sécuriser les sessions                 |
+| Variable                 | Valeur par défaut        | Rôle                                                       |
+| ---                      | ---                      | ---                                                        |
+| `GO_ENV`                 | `development`            | L'« environment » d'exécution de Buffalo                   |
```
Comment (lmazuel) [inline templates/docs/config-vars.fr.md]: **L’environnement** d'exécution de Buffalo

File: templates/docs/config-vars.fr.md [hunk 2]
```diff
@@ -11,15 +11,16 @@ Les variables d'environnement sont une bonne méthode pour séparer les valeurs
...
+| `GO_BIN`                 | `go`                     | Le compilateur Go à utiliser                               |
+| `BUFFALO_PLUGIN_PATH`    | `$PATH`                  | L'endroit où Buffalo va chercher ses plugins               |
```
Comment (lmazuel) [inline templates/docs/config-vars.fr.md]: **L'emplacement** où Buffalo va chercher ses plugins
(not critical but more fluent I'd say. some might disagree)

File: templates/docs/config-vars.fr.md [hunk 3]
```diff
@@ -11,15 +11,16 @@ Les variables d'environnement sont une bonne méthode pour séparer les valeurs
...
+| `BUFFALO_PLUGIN_TIMEOUT` | `1s`                     | Combien de temps Buffalo attend pour un plugin à répondre  |
```
Comment (lmazuel) [inline templates/docs/config-vars.fr.md]: Combien de temps Buffalo attend **la réponse d'un plugin**

**Comment by stanislas-m** [comment]
Thanks!

**Pull request merged**